pixart.js

```diff
@@ -1 +1,37 @@
+var button = $("#button");
+// var input = $(".text");
+var brush = $(".brush");
 
+function changeBrushColortoInput () {
+  var color = document.getElementById("color-field").value;
+  // var color = $("#color-field").val(); //need to get the value from form
+  $(".brush").css("background","color");
```

Comment:
This will change the brush's background to the string "color" -- not an actual color.

You need to replace `"color"` with a reference to whatever the user has entered into the input field. Fortunately, you've already stored that in `var color`. You just need to use it on line 8.